index.js

```diff
@@ -1,5 +1,5 @@
 function main(){
-// edits coming
+

```

Comment:
I do not see the sections of code that I am trying to revert being highlighted. Hopefully they will actually be reverted.